torch/csrc/utils/python_symnode.h

```diff
@@ -95,6 +106,23 @@ class PythonSymNodeImpl : public c10::SymNodeImpl {
     return getPyObj().attr("is_bool")().is(py::handle(Py_True));
   }
 
+  bool is_singleton() const override {
+    return is_singleton_;
+  }
+
+  c10::TensorImpl* singleton_vec() const override {
+    py::gil_scoped_acquire acquire;
+    return getPyObj()
+        .attr("singleton_vec")()
+        .cast<at::Tensor>()
+        .unsafeGetTensorImpl();
+  }
+
+  int64_t singleton_sum_vec() const override {
```

Comment:
I think this needs to return a SymInt, not an int64_t, to handle fake tensor

Reply:
No longer storing sum_vec on the nested int